.gitignore

```diff
@@ -1,15 +1,17 @@
 /.bundle/
-/.idea/
```

Comment:
@agrare we didn't stop to using RubyMine... ;)

Reply:
This is the standard .gitignore from swagger, have you tried a global gitignore?  I usually don't like adding ide/editor specifics into the project since everyone uses something different and the union would be huge.

Reply:
I didn't know about global gitignore, I'll try, :+1:

Reply:
I have `.byebug_history` in mine because I've accidentally committed that a few too many times with `git add . && git commit --amend` :D

Reply:
yeah, I rather stopped using `git add .`  :smile: 
Anyway thanks for tip